lib/entries.rb

```diff
@@ -1,15 +1,79 @@
 require 'pry'
+require_relative 'symbols.rb'
 
 class Entries
 
-  def initialize(account_number)
-    @account_number
-    @account_number_display = []
+  def initialize
+    @symbols = Symbols.new
+    @entry = 0
+    @entries_as_symbols = [[],[],[]]
+    @account_numbers = []
   end
 
-  attr_accessor :account_number
+  attr_accessor :entry, :account_numbers
+  attr_reader :symbols, :entries_as_symbols
 
-  def display_digits(account_number)
-    account_number_array = account_number.split('')
+
+# code to validate entry before passing it to symbols.rb - does not work.
+  # def entry_validation(entry)
+  #   entry_number_of_digits(entry)
+  #   true
+  # rescue
+  #   puts 'Exception intercepted. 9 digit number required.'
+  #   false
+  # end
+  #
+  # def entry_number_of_digits(entry)
+  #   return true if entry.entry.to_s.chars.map(&:to_i).count != 9
+  # end
+
+#User story 3 solution
+  def print_account_numbers
+    ac_nmb = get_account_numbers
+    ac_nmb.each do |subarray|
+      if contains_question_mark(subarray) == true
+        a = subarray.join('') + ' ILL'
+        puts a
+      elsif is_valid(subarray) == true
+        a = subarray.join('')
+        puts a
+      elsif is_valid(subarray) == false
+         a = subarray.join('') + ' ERR'
+         puts a
+      end
+    end
+  end
+
+  def contains_question_mark(subarray)
```

Comment:
is this a boolean method? what should this return if none of the items include a '?'